src/containers/cats/cats.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { onRandomCat } from 'redux/actions/cats'
+
+import RandomCat from 'components/cats/cat'
```

Comment:
We have changed the name of the component here from `Cat` => `RandomCat`. I would pick one and stick with it across this whole PR